charts/zookeeper-operator/templates/crd.yaml

```diff
@@ -1,47 +1,6 @@
 {{- if .Values.crd.create }}
```

Comment:
This file addition is unnecessary. I feel u should revert this to your previous change and remove the option for user to specify whether the crd needs to be created from the values.yaml. As per my understanding from [this](https://github.com/helm/helm/pull/6243) PR :
```
If a CRD is already found in the cluster, Helm will skip installing that CRD and will log this to the debug log. It will neither error out nor see if there are any differences between its version and the existing version.
```

Reply:
Okay I will remove it then. I was thinking that this could be used for backward compatibility with helm v2.

Reply:
We have migrated to helm 3 now, and don't plan to support helm 2